index.js

```diff
@@ -80,7 +80,7 @@ function fastifyStatic (fastify, opts, next) {
   if (opts.prefix === undefined) opts.prefix = '/'
   const prefix = opts.prefix[opts.prefix.length - 1] === '/' ? opts.prefix : (opts.prefix + '/')
 
-  fastify.get(prefix + '*', function (req, reply) {
+  fastify.get(prefix + '*', { schema: { hide: true } }, function (req, reply) {
```

Comment:
can you please add a comment here explaining why we are doing this?

Reply:
done in the newest changes. If need something else, please advice me.